src/scanner/ScannerUtils.ts

```diff
@@ -99,4 +102,30 @@ export class ScannerUtils {
 
     return { customApplicablePractices, practicesOff };
   }
+
+  /**
+   * Get all levels to fail on
+   */
+  static getImpactFailureLevels = (impact: PracticeImpact | 'all' | undefined) => {
+    switch (impact) {
+      case PracticeImpact.high:
+        return [PracticeImpact.high];
+      case PracticeImpact.medium:
+        return [PracticeImpact.high, PracticeImpact.medium];
+      case PracticeImpact.small:
+        return [PracticeImpact.high, PracticeImpact.medium, PracticeImpact.small];
+      case PracticeImpact.hint:
+        return [PracticeImpact.high, PracticeImpact.medium, PracticeImpact.small, PracticeImpact.hint];
+      default:
+        return [];
+    }
+  };
+
+  static filterNotPracticingPracticesToFail = (reportArguments: PracticeWithContextForReporter[], argumentsProvider: ArgumentsProvider) => {
```

Comment:
All functions in this class are documented and here is missing the documentation.